test/screens/v2/candidate_generator/pre_fare_test.exs

```diff
@@ -55,4 +57,35 @@ defmodule Screens.V2.CandidateGenerator.PreFareTest do
               }} == PreFare.screen_template()
     end
   end
+
+  describe "candidate_instances/3" do
+    test "returns expected header", %{config: config} do
+      now = ~U[2020-04-06T10:00:00Z]
+      elevator_status_instances_fn = fn _, _ -> [] end
+
+      expected_header = [
+        %NormalHeader{
+          screen: config,
+          icon: nil,
+          text: "Test Station",
+          time: ~U[2020-04-06T10:00:00Z]
+        },
+        %NormalHeader{
+          screen: config,
+          icon: nil,
+          text: "Test Station",
+          time: ~U[2020-04-06T10:00:00Z]
+        }
```

Comment:
Are two header instances still expected now that we no longer separate left vs. right?

Reply:
Aha!  Thanks!
```suggestion
        %NormalHeader{
          screen: config,
          icon: nil,
          text: "Test Station",
          time: ~U[2020-04-06T10:00:00Z]
        }
```